sources/Interop/Windows/um/winnt/ACTIVATION_CONTEXT_COMPATIBILITY_INFORMATION.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/winnt.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct ACTIVATION_CONTEXT_COMPATIBILITY_INFORMATION
+    {
+        [NativeTypeName("DWORD")]
+        public uint ElementCount;
+
+        [NativeTypeName("COMPATIBILITY_CONTEXT_ELEMENT []")]
+        public COMPATIBILITY_CONTEXT_ELEMENT* Elements;
```

Comment:
I checked the MSVC type layout and this is still inline. That is, it should be interpreted as a `fixed sized buffer` of potentially length 0

Reply:
I believe these are the cases we talked about on Discord. Originally ClangSharp was generating just `public COMPATIBILITY_CONTEXT_ELEMENT Elements;` I talked about it with you and you thought that making it a pointer was okay. I'm not sure what code you actually expect to see here.

Reply:
We agreed ClangSharp needs to be improved to handle zero-length arrays.